.eslintrc.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
                         tsx: 'never',
                     },
                 ],
+                'rulesdir/prefer-onyx-connect-in-libs': 'off',
```

Comment:
[DUMB QUESTION] Why exactly did we turn that off here?

Reply:
This rule is complaining in my TS files because I'm calling `Onyx` methods

Reply:
@tgolen Actually I just saw that [we are suppressing the rule in withOnyx.js too](https://github.com/Expensify/react-native-onyx/blob/main/lib/withOnyx.js#L288). I don't think is a problem with the ESLint, just that it doesn't make too much sense to have this rule enabled in this repository since `rulesdir/prefer-onyx-connect-in-libs` looks like tailored for E/App. For example, this is the error that is shown:

> Only call Onyx.connect() from inside a /src/libs/** file. React components and non-library code should not use Onyx.connect()

It doesn't apply to here.

Reply:
OK, thanks! I guess it might make more sense if the rule was turned off by default and then only enabled in the App repo, since it's very specific to that repo. I'll have to keep this in mind for future custom ESLint rules.